spec/inputs/file_spec.rb

```diff
@@ -8,11 +8,11 @@
 
   delimiter = (LogStash::Environment.windows? ? "\r\n" : "\n")
 
-  describe "starts at the end of an existing file" do
+  it "should starts at the end of an existing file" do
     tmpfile_path = Stud::Temporary.pathname
     sincedb_path = Stud::Temporary.pathname
 
-    config <<-CONFIG
+    conf = <<-CONFIG
```

Comment:
Ok, go it .... I found this kind  of error prone to keep, but for now I don't see a very important reason to rip it off.